### PR TITLE
Set up nightly run of window fuzzer

### PR DESCRIPTION
**Pull request by kagamiori**
Differential Revision: D55027049

### CODE REVIEW

File: .github/workflows/experimental.yml [hunk 1]
```diff
@@ -171,6 +171,72 @@ jobs:
...
+          path: velox
+          submodules: 'recursive'
+          ref: "${{ inputs.ref || 'main' }}"
+
+
+      - name: "Build"
+        run: |
+          cd velox
+          source /opt/rh/gcc-toolset-9/enable
+          make debug NUM_THREADS="${{ inputs.numThreads || 8 }}" MAX_HIGH_MEM_JOBS="${{ inputs.maxHighMemJobs || 8 }}" MAX_LINK_JOBS="${{ inputs.maxLinkJobs || 4 }}" EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON ${{ inputs.extraCMakeFlags }}"
```
Comment (kgpai) [inline .github/workflows/experimental.yml:206]: change the highmen jobs and to also 4

File: .github/workflows/scheduled.yml [hunk 2]
```diff
@@ -810,3 +817,66 @@ jobs:
...
+          sleep 60
+          /opt/presto-cli --server 127.0.0.1:8080 --execute 'CREATE SCHEMA hive.tpch;'
+          cd -
+          mkdir -p /tmp/window_fuzzer_repro/
+          rm -rfv /tmp/window_fuzzer_repro/*
+          chmod -R 777 /tmp/window_fuzzer_repro
+          chmod +x velox_window_fuzzer_test
+          ./velox_window_fuzzer_test \
+                --seed ${RANDOM} \
+                --duration_sec 3600 \
```
Comment (kgpai) [inline .github/workflows/scheduled.yml:867]: use $DURATION

**Review by kgpai — COMMENTED** [review]
Changes to scheduled.yml look fine to me , save one minor comment. Can you get @aditi-pandit to look at change in Window.cpp

**Comment by kagamiori** [comment]
> Changes to scheduled.yml look fine to me , save one minor comment. Can you get @aditi-pandit to look at change in Window.cpp

Hi @kgpai, the first commit that includes changes in Window.cpp is actually another PR https://github.com/facebookincubator/velox/pull/9476. The current PR is based on top of that, so it includes both commits. Sorry for the confusion.

File: .github/workflows/scheduled.yml (hunk 2 above)
Comment (kagamiori) [reply .github/workflows/scheduled.yml]: Updated. Thanks!

**Review by kgpai — APPROVED** [review]

**Pull request closed without merging**